src/AutoRest.CSharp/MgmtTest/Generation/ResourceTestWriter.cs

```diff
@@ -110,10 +107,12 @@ protected void WriteTesterCtors()
             }
         }
 
-        public string WriteGetResource(Resource resource, string requestPath, ExampleModel exampleModel)
+        public string WriteGetResource(Resource resource, string resourceIdentifierParams, ExampleModel exampleModel)
```

Comment:
I believe this parameter `resourceIdentifierParams`, its actual value is like `"0000-0000-...", "myRg", "myAvailabilitySet"`?
I suggest we change this type to `FormattableString` which is a recommended way to pass a intermediate result of the code writer, which leaves some space for the case that we need some special format that only code writer could handle.

Reply:
yes, the resourceIdentifierParams does value to string like ("0000-0000-...", "myRg"..).
I will change the type.